Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppleMDMPushNotificationCertificate/MSFT_IntuneAppleMDMPushNotificationCertificate.schema.mof

```diff
@@ -0,0 +1,25 @@
+[ClassVersion("1.0.0.0"), FriendlyName("IntuneAppleMDMPushNotificationCertificate")]
+class MSFT_IntuneAppleMDMPushNotificationCertificate : OMI_BaseResource
+{
+    [Key, Description("The name of the Apple Identifier.")] String AppleIdentifier;
+    [Write, Description("The Apple Push notification certificate.")] String Certificate;
+
+    [Write, Description("The unique identifier for an entity. Read-only.")] String Id;
+    [Write, Description("The Topic Identifier. Also known as Subject Id.")] String TopicIdentifier;
+    [Write, Description("The upload status of the Apple Push notification certificate.")] String CertificateSerialNumber;
```

Comment:
Can these properties be set outside of the certificate payload? Meaning that if in the certificate it says "Value X", you can here say "Value Y", and it will still be reflected by the API? I have my doubts, but I'm unable to test. 
Also in `Set-TargetResource`, these are removed before executing the Graph cmdlet.